regparser/tree/xml_parser/appendices.py

```diff
@@ -339,7 +358,7 @@ def parsed_title(text, appendix_letter):
                         + Optional(grammar.paren_upper | grammar.paren_lower)
                         + Optional(grammar.paren_digit))
     part_roman_parser = Marker("part") + grammar.aI
-    parser = LineStart() + (digit_str_parser | part_roman_parser)
+    parser = LineStart() + (digit_str_parser | part_roman_parser | grammar.roman_upper)
```

Comment:
See comment above, but it's not clear to me how using `grammar.roman_upper` gets anything different from `part_roman_parser`, which uses `grammar.aI`.

Tests would be really useful in illustrating this change.